benchmark/src/index.ts

```diff
@@ -179,8 +163,8 @@ function cachedBuildWithDependencyChange(): Timing[] {
 }
 
 class Benchmarks {
-  private readonly benchmarks: Benchmark[] = [];
-  private readonly tbirdEvents: TBirdEvent[] = [];
+  private readonly benchmarks: Array<Benchmark> = [];
```

Comment:
Just curious, is there a reason for these changes? Like a lint or best practice or something?

Reply:
Since I was reusing some of the code from `src/index.ts`, I added tsc and eslint checking because I kept messing up easy things that tooling can catch. I believe some of these changes are from `eslint --fix`. I didn't spendt oo much time separating because it's not production code